src/test/java/com/wikia/webdriver/common/core/url/BaseUrlBuilder.java

```diff
@@ -0,0 +1,53 @@
+package com.wikia.webdriver.common.core.url;
+
+import static com.wikia.webdriver.common.core.configuration.Configuration.getEnvType;
+
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.core.configuration.EnvType;
+import org.apache.commons.lang.StringUtils;
+
+public class BaseUrlBuilder {
+
+    protected static final String HTTP_PREFIX = "http://";
+    protected String env;
+    protected EnvType envType;
+
+    public BaseUrlBuilder(String env) {
+        this.env = env;
+        this.envType = getEnvType(this.env);;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)